### ISSUE TITLE
Tasks - can there be a from time

### ISSUE THREAD

**Issue by rarvin** [kevinpapst/kimai-plugins#13]
Could you help create some task where the from time could be assigned in advance or it could be a said duration before the to time which is configurable (also having a default value). The tasks need not start exactly from the time time, but are an indicator that they should start by them. If they do not start by then they are to be flagged as amber or red.

Also can one task be linked to another task indicating the second is dependent on the first and if the first breaches the to time and this time is greater than the from start time for the second task them its a breach and tuns the second task also into red. Chaining or linking could be a powerful feature. If chaining of tasks is not set then they are independent tasks capable of being executed in parallel or sequentially as required.

Also can there be a setting available like a set amount of time configurable (and a default value if to time is set) by when if any task has breached the threshold the status turns to amber indicating if action is not taken then the task would fall into jeopardy of not being able to meet the final deadline.
